### DISCUSSION TITLE
Where is the English text source file?

### DISCUSSION THREAD

**Discussion by ViBE-HU** [input-leap/input-leap#1735] [Development of Input Leap]
I'm not following actively the project nowadays. But I see that sometimes the repo got some updates and It's been a long time since I checked my translation since it is based on the original Barrier language files. So I would review the translation but I cannot find the English source (.ts). Am I blind or it does not exist?

**Comment by shymega** [comment]
Should be `src/gui/res/lang`.

**Comment by ViBE-HU** [comment]
I downloaded the repo and checked several ways. But no luck. No "en" file.

**Comment by shymega** [comment]
Misread. I don't think we've ever had an `en` file. Or maybe it was refactored. But you should see it in the original QML markup. Each translation in `src/gui/res/lang` is going to override the original text in QML.

**Comment by ViBE-HU** [comment]
I know. But I dunno translation updates are automatized or not and I was suspicious that may some strings missing.

**Reply by shymega** [reply]
It's not automated. We could use Weblate in future though.

**Reply by ViBE-HU** [reply]
That would be nice too.
